### ISSUE TITLE
Up/down chevrons on the dropdown form control disappears in Windows HCM 

### ISSUE THREAD

**Issue by trimspa** [uswds/uswds#4454]
### Describe the bug
The up/down chevrons on the dropdown form control disappear in Windows HCM

### Steps to reproduce the bug
1. Turn on Windows High Contrast Mode.
2. Go to the [Dropdown](https://designsystem.digital.gov/components/dropdown/) component on the USWDS site
2. Scroll to the first dropdown example
3. Chevron on the form control is missing 

### Expected behavior
Given that this chevron conveys visual meaning that aids user navigation, it should be visible.

### Screenshots
<img width="848" alt="Screen Shot 2022-01-07 at 4 13 34 PM" src="https://user-images.githubusercontent.com/64163603/148614366-3fe2a291-19c6-4d29-a42e-2d9e9406ee27.png">

### System setup
- **USWDS version:**
- **Device: MacBook Pro**
- **Operating system: Windows 10 (Parallels), Big Sur 11.6**
- **Browser and version: Edge**

### Additional context

We/I will be filing a PR for this issue next week. Just wanted to add the issue here for tracking purposes.
